pkg/config/config.go

```diff
@@ -82,11 +82,19 @@ func getKernelVersion(c Client) float32 {
 	}
 	// per https://github.com/google/cadvisor/blob/master/machine/info.go#L164
 	kv := utsname.Release[:bytes.IndexByte(utsname.Release[:], 0)]
+	// for something like 4.18.10-24-generic
 	val, err := strconv.ParseFloat(string(kv[:4]), 32)
 	if err == nil {
 		return float32(val)
+	} else {
+		// for something like 5.4.0-48-generic
```

Comment:
`5.4.0-48-generic` and `4.18.10-24-generic` looks very similar to me
What is the problem here?

Reply:
check https://github.com/sustainable-computing-io/kepler/issues/179#issuecomment-1253069991

Reply:
what about kernel `5.0, 6.0`?
https://en.wikipedia.org/wiki/Linux_kernel_version_history

how about splitting the string by `.`,  taking the first two substrings, and recreating a new kernel string?